src/styles/reset.scss

```diff
@@ -0,0 +1,21 @@
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
do not reset margin and padding for all selectors
it is better to make just for elements which really need it
`*` selector in general only for box-sizing property
```suggestion
```